### PR TITLE
[release] 4.10 Release and Blog post

### PR DESCRIPTION
**Pull request by romainr**
- [release] 4.10 Blog post

### CODE REVIEW

File: docs/gethue/content/en/posts/2021-06-14-release-hue-4-10.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+categories:
+  - Version 4.10
+  - Release
+
+---
+Hi Data Explorers,
+
+The Hue Team is glad to thanks all the contributors and release Hue 4.10!
+
+<a href="https://cdn.gethue.com/uploads/2021/02/hue-4.9.png">
```
Comment (JohanAhlen) [inline docs/gethue/content/en/posts/2021-06-14-release-hue-4-10.md:50]: Could be on purpose, but img from 4.9.

File: docs/gethue/content/en/posts/2021-06-14-release-hue-4-10.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  <img src="https://cdn.gethue.com/uploads/2021/02/hue-4.9.png" />
+</a>
+
+&nbsp;
+
+Here is a summary of the [main improvements](/categories/version-4.10/) of 4.10 on top of the previous [4.9](/blog/hue-4-9-sql-dialects-phoenix-dasksql-flink-components/) release:
+
+
+## SQL Editor Component & API
+Now build your own SQL Editor in two HTML line by leveragin the [SQL Scratchpad](https://docs.gethue.com/developer/components/scratchpad/) component. The first version of the public [REST API](https://docs.gethue.com/developer/api/rest/) for executing queries was published to.
```
Comment (JohanAhlen) [inline docs/gethue/content/en/posts/2021-06-14-release-hue-4-10.md:60]: "line by levragin" => "lines by leveraging" and "queries was published to" => "queries was published too"

File: docs/gethue/content/en/posts/2021-06-14-release-hue-4-10.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+Read more about the [Sql Scratchpad and REST API](https://gethue.com/blog/2021-05-29-create-own-sql-editor-via-webcomponent-and-public-api/).
```
Comment (JohanAhlen) [inline docs/gethue/content/en/posts/2021-06-14-release-hue-4-10.md:62]: "Sql Scratchpad" => "SQL Scratchpad"

File: docs/gethue/content/en/posts/2021-06-14-release-hue-4-10.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+Here is a live demo of how easy it is to add the component:
+
+{{< rawhtml >}}
+<p>
+  <div style="position: absolute; height: 10%; width: 100%">
```
Comment (JohanAhlen) [inline docs/gethue/content/en/posts/2021-06-14-release-hue-4-10.md:68]: height: 100%

File: docs/gethue/content/en/posts/2021-06-14-release-hue-4-10.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+    <sql-scratchpad api-url="https://demo.gethue.com" username="demo" password="demo" dialect="mysql" />
+  </div>
+
+  <script type="text/javascript" src="https://unpkg.com/gethue/lib/components/SqlScratchpadWebComponent.js"></script>
+
+  <br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
```
Comment (JohanAhlen) [inline docs/gethue/content/en/posts/2021-06-14-release-hue-4-10.md:74]: `<br>` needed?

File: docs/gethue/content/en/posts/2021-06-14-release-hue-4-10.md (hunk 2 above)
Comment (JohanAhlen) [reply docs/gethue/content/en/posts/2021-06-14-release-hue-4-10.md:60]: "two HTML line" we say three below.

**Review by JohanAhlen — APPROVED** [review]

**Comment by JohanAhlen** [comment]
Looks good! Nice! Just some minor spelling adjustments.

File: docs/gethue/content/en/posts/2021-06-14-release-hue-4-10.md (hunk 1 above)
Comment (romainr) [reply docs/gethue/content/en/posts/2021-06-14-release-hue-4-10.md:50]: Yes, 4.10 not cut yet!

File: docs/gethue/content/en/posts/2021-06-14-release-hue-4-10.md (hunk 4 above)
Comment (romainr) [reply docs/gethue/content/en/posts/2021-06-14-release-hue-4-10.md:68]: It is a bit flaky currently: then it expands to all the page, I can show you in person the <br> and https://github.com/cloudera/hue/pull/2222

File: docs/gethue/content/en/posts/2021-06-14-release-hue-4-10.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+Read more about the [create table from a small file](https://gethue.com/blog/2021-05-26-improved-hue-importer-select-a-file-choose-a-dialect-create-a-table/).
+
+![Importer direct upload steps](https://cdn.gethue.com/uploads/2021/05/Importer_direct_upload_steps.gif)
+
+## Slack App
+Collaborate more via Slack thanks to the rich preview of SQL queries, auto links, query bank and export result to Slack.
+
+[Install the app](/blog/2021-05-18-installing-hue-slack-app-in-three-simple-steps/) in a few clicks and learn more about the [functionalities of assistance](https://docs.gethue.com/user/concept/#slack).
+
+![Slack Installation Flow](https://cdn.gethue.com/uploads/2021/05/slack-install.gif)
```
Comment (Harshg999) [inline docs/gethue/content/en/posts/2021-06-14-release-hue-4-10.md:93]: This gif is also present in the `Install the app` blog. Can we show the easier installation gif? Something like

`Can also be installed within seconds by just giving your Hue hostname and  following the [steps] (https://docs.gethue.com/administrator/configuration/server/#improved-slack-app-installation)`

```Slack Installation Flow```

**Review by Harshg999 — APPROVED** [review]
Great overall summary!

**Pull request merged**